CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## [1.0.2] - 2021-09-02
+
+### Added
+- Experimental support for custom authorization policies based on Open Policy Agent (OPA) (#2416)
+- SPIRE Server can now be configured to emit audit logs (#2297, #2391, #2394, #2396, #2442, #2458)
+- SDS v3 API in agent now supports the SPIFFE Certificate Validator (#2435, #2460)
+- Enhancements in the SPIRE OIDC Discovery Provider (#2404, #2453)
```

Comment:
There were a couple changes with these PRs but I think these two were the main ones.

```suggestion
- SPIRE OIDC Discovery Provider can now listen on a unix domain socket (#2404)
- SPIRE OIDC Discovery Provider now intelligently handles host headers (#2404, #2453)
```

Reply:
@amartinezfayo pointed out that the provider already supported UDS

```suggestion
- SPIRE OIDC Discovery Provider now intelligently handles host headers (#2404, #2453)
```